helm_deploy/prisoner-content-hub-backend/templates/ingress-jsonapi-old.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "prisoner-content-hub-backend.fullname" . -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}-jsonapi
+  labels:
+    {{- include "prisoner-content-hub-backend.labels" . | nindent 4 }}
+  {{- with .Values.ingress.jsonapi.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+    nginx.ingress.kubernetes.io/whitelist-source-range: {{ include "app.joinListWithComma" .Values.ingress.jsonapi.allowed | quote }}
+    kubernetes.io/ingress.class: "nginx"
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
+spec:
+  tls:
+  {{- range .Values.ingress.jsonapi.hosts }}
+  - hosts:
+    - {{ .host }}
+    {{ if .cert_secret }}secretName: {{ .cert_secret }}{{ end }}
+  {{- end }}
+  rules:
+  {{- range .Values.ingress.jsonapi.hosts }}
+    - host: {{ .host | quote }}
+      http:
+        paths:
+          - path: "/"
+            backend:
+              serviceName: {{ $fullName }}
+              servicePort: http
+  {{- end }}
+{{- end -}}
```

Comment:
This file is a copy of the previous ingress.yml file, but is now labelled "-old" as it will be removed once all this is deployed